### PR TITLE
Merge release 22.11 into master

### PR DESCRIPTION
**Pull request by freddyucv**

### CODE REVIEW

File: .github/workflows/release-process.yml [hunk 1]
```diff
@@ -7,7 +7,7 @@ jobs:
     name: Release Process Automation
     runs-on: ubuntu-latest
     env:
-      DOT_CICD_BRANCH: master
+      DOT_CICD_BRANCH: release-22.11
```
Comment (fmontes) [inline .github/workflows/release-process.yml:10]: master

File: .gitmodules [hunk 2]
```diff
@@ -1,5 +1,5 @@
 [submodule "dotCMS/src/main/enterprise"]
 	path = dotCMS/src/main/enterprise
 	url = git@github.com:dotCMS/enterprise.git
-	branch = master
+	branch = release-22.11
```
Comment (fmontes) [inline .gitmodules:4]: master

File: docker/docker-compose-examples/with-redis/docker-compose-node-1.yml [hunk 3]
```diff
@@ -37,6 +37,19 @@ services:
   dotcms-node-1:
     image: dotcms/dotcms:latest
     environment:
+<<<<<<< HEAD
```
Comment (fmontes) [inline docker/docker-compose-examples/with-redis/docker-compose-node-1.yml:40]: merge issues.

File: dotCMS/gradle.properties [hunk 4]
```diff
@@ -1,6 +1,6 @@
-dotcmsReleaseVersion=master
-coreWebReleaseVersion=next
-webComponentsReleaseVersion=next
```
Comment (fmontes) [inline dotCMS/gradle.properties:3]: This is what we need in master.

**Review by fmontes — CHANGES_REQUESTED** [review]

File: .github/workflows/release-process.yml (hunk 1 above)
Comment (freddyucv) [reply .github/workflows/release-process.yml]: done https://github.com/dotCMS/core/pull/23368/commits/2b4582e8d7ff6adaca5c671b58a6f2ef8704b867#diff-f043e10450bb99dcc10b694d4c5618fceff2763571abb8dae6142774bf4fb40cR10

File: .gitmodules (hunk 2 above)
Comment (freddyucv) [reply .gitmodules]: done https://github.com/dotCMS/core/pull/23368/commits/2b4582e8d7ff6adaca5c671b58a6f2ef8704b867#diff-fe7afb5c9c916e521401d3fcfb4277d5071798c3baf83baf11d6071742823584R4

File: docker/docker-compose-examples/with-redis/docker-compose-node-1.yml (hunk 3 above)
Comment (freddyucv) [reply docker/docker-compose-examples/with-redis/docker-compose-node-1.yml]: done https://github.com/dotCMS/core/pull/23368/commits/2b4582e8d7ff6adaca5c671b58a6f2ef8704b867#diff-dae0b9b8d9233e5b2abbe45229a0042d2a866c7c93e045de40c1b36fb13a2ab5L40

File: dotCMS/gradle.properties (hunk 4 above)
Comment (freddyucv) [reply dotCMS/gradle.properties]: done https://github.com/dotCMS/core/pull/23368/commits/2b4582e8d7ff6adaca5c671b58a6f2ef8704b867#diff-01eec14c9d36ad794af257b0869db9c6525d69417c306a7cd2cd9dae64c9710bR1

**Review by fmontes — APPROVED** [review]

File: core-web/package-lock.json [hunk 5]
```diff
@@ -1,12 +1,12 @@
 {
   "name": "dotcms-ui",
-  "version": "22.12.0-next.15",
+  "version": "22.12.0-next.16",
```
Comment (dsilvam) [inline core-web/package-lock.json:3]: leave the value on master

File: core-web/package-lock.json [hunk 6]
```diff
@@ -1,12 +1,12 @@
...
   "lockfileVersion": 2,
   "requires": true,
   "packages": {
     "": {
       "name": "dotcms-ui",
-      "version": "22.12.0-next.15",
+      "version": "22.12.0-next.16",
```
Comment (dsilvam) [inline core-web/package-lock.json:9]: leave the value on master

**Review by dsilvam — APPROVED** [review]

**Pull request merged**